code/src/layouts/root-layout/RootLayout.scss

```diff
@@ -0,0 +1,9 @@
+.root-layout {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
```

Comment:
maybe use dvh property? Also, we can add this to our scss variable screen-height and screen-width or smth like that. What do you think?

Reply:
Thnak you. I used `svh` for it because it is static, and with `dvh` there can be unexpected resizes and other bugs. I also added a variable in design system and used it.